src/GovUk.Education.ExploreEducationStatistics.Common/Model/Chart/ChartAxisConfiguration.cs

```diff
@@ -73,6 +73,9 @@ public class AxisReferenceLine
         public AxisReferenceLineStyle? Style;
 
         public int? OtherAxisPosition;
+
+        public string? StartPosition;
```

Comment:
These two new fields should follow the same code style as other fields in the model (1 blank line between them)